<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailWindow</class>
 <widget class="QWidget" name="EmailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>625</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/**
 * Blue: rgb(61, 149, 245);
 * Hover Blue: rgb(13, 122, 242);
 * Light Hover Blue: rgb(187, 225, 250);
 * Red: rgb(242, 22, 43);
 * Hover Red: rgb(203, 11, 29);
 * Green: rgb(82, 224, 102);
 * Yellow: rgb(253, 214, 52);
 * Hover Yellow:  rgb(253, 205, 36);
 * Hover Green: rgb(39, 216, 63);
 * Light Gray: rgb(200, 200, 200);
 * Gray : rgb(239, 239, 239);
 * White: rgb(248, 248, 248);
 */

#EmailWindow {
	background-color: rgb(239, 239, 239);
	border-radius: 10px;
}

QLabel {
	background-color: rgb(239, 239, 239);
}

QLineEdit {
	background-color: rgb(248, 248, 248);
	border-radius: 5px;
	border: 2px solid rgb(200, 200, 200);
}

QLineEdit:hover {
	border: 2px solid rgb(187, 225, 250);
}

QLineEdit:focus {
	border: 2px solid rgb(13, 122, 242);
}

QSpinBox {
	background-color: rgb(248, 248, 248);
	border-radius: 5px;
	border: 2px solid rgb(200, 200, 200);
}

QSpinBox:hover {
	border: 2px solid rgb(187, 225, 250);
}

QSpinBox:focus {
	border: 2px solid rgb(13, 122, 242);
}

QPlainTextEdit {
	background-color: rgb(248, 248, 248);
	border-radius: 5px;
	border: 2px solid rgb(200, 200, 200);
}

QPlainTextEdit:hover {
	border: 2px solid rgb(187, 225, 250);
}

QPlainTextEdit:focus {
	border: 2px solid rgb(13, 122, 242);
}

QPushButton {
	background-color: rgb(61, 149, 245);
	color: rgb(248, 248, 248);
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(13, 122, 242);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QLabel" name="ccLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CC:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="receiverLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>example@mail.com</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="idSpinBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="subjectLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Hello</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="receiverLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Receiver:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="idLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="subjectLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Subject:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLineEdit" name="bccLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>1020</number>
     </property>
     <property name="placeholderText">
      <string>Emails should be separated by a semicolon (;)</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="bccLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BCC:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLineEdit" name="ccLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>1020</number>
     </property>
     <property name="placeholderText">
      <string>Emails should be separated by a semicolon (;)</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPlainTextEdit" name="contentLineEdit">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Your email content goes here</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QPushButton" name="writeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/send.png</normaloff>:/icons/send.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="senderLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sender:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="senderLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>example@mail.com</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="datetimeFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="dateLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>idSpinBox</tabstop>
  <tabstop>subjectLineEdit</tabstop>
  <tabstop>senderLineEdit</tabstop>
  <tabstop>receiverLineEdit</tabstop>
  <tabstop>ccLineEdit</tabstop>
  <tabstop>bccLineEdit</tabstop>
  <tabstop>contentLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
